lib/mixin/sentry_state_transaction_mixin.dart

```diff
@@ -0,0 +1,21 @@
+library after_layout;
+
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+mixin SentryStateTransactionMixin<T extends StatefulWidget> on State<T> {
+  ISentrySpan? transaction;
+
+  @override
+  void initState() {
+    super.initState();
+    transaction = Sentry.startTransaction(
+      runtimeType.toString(),
+      'ui.load',
+      bindToScope: true,
+    );
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
```

Comment:
if `WidgetsBinding.instance` is null, `transaction` never finishes and leaks memory.

does `addPostFrameCallback` gets called for every screen? after rendering or what?

Reply:
At this point `WidgetsBinding.instance` can never be null.

Reply:
Does it make sense to check anyway?

Reply:
No, not really. There's already a PR in the Flutter repo somewhere which makes `WidgetsBinding.instance` non-nullable in a future version.